### DISCUSSION TITLE
How to configure Microcks with Apicurio Registry

### DISCUSSION THREAD

**Discussion by marcin-brzozowski** [microcks/microcks#637] [Q&A] [answered]
According to the [documentation](https://microcks.io/documentation/guides/avro-messaging/) Microcks has been successfully tested with `Apicurio Registry`.

`Microcks has been successfully tested with both [Confluent Schema Registry](https://github.com/confluentinc/schema-registry) and [Apicurio Registry](https://www.apicur.io/registry/). Both can be deployed as containerized workload on your Kubernetes cluster.`

Does anyone know of any resource that could help me set deploy and configure Microcks with Apicurio Registry? I was looking through the helm charts and operator, but couldn't find anything there.

Thank you in advance!

**Comment by lbroudoux** [comment, marked as answer]
Hey there! Have you checked our guide here: https://microcks.io/documentation/guides/avro-messaging/#1-setup-schema-registry ?
